cartofacade/capi.go

```diff
@@ -74,6 +75,16 @@ type GetPosition struct {
 	ComponentReference string
 }
 
+// IMUReading holds values for linear acceleration and angular velocity to be converted into c
+type IMUReading struct {
+	LinAccX float64
```

Comment:
Let's make these r3 Vectors

Reply:
`r3.Vector` and `spatialmath.AngularVelocity`

Reply:
Done